src/Symfony/Component/Finder/Tests/Iterator/RecursiveDirectoryIteratorTest.php

```diff
@@ -73,7 +70,6 @@ public function getPaths()
         // ftp
         $contains = array(
             'ftp://ftp.mozilla.org'.DIRECTORY_SEPARATOR.'README',
-            'ftp://ftp.mozilla.org'.DIRECTORY_SEPARATOR.'index.html',
```

Comment:
Looks like we now test this (it is skipped on travis) and Mozilla does not publish this file anymore.